core/src/components/textarea/test/highlight/index.html

```diff
@@ -174,6 +181,49 @@ <h2>Invalid</h2>
           </div>
         </div>
 
+        <h1>No Fill, Custom</h1>
+        <div class="grid">
+          <div class="grid-item">
+            <h2>Focus</h2>
+            <ion-textarea
+              label-placement="start"
+              value="hi@ionic.io"
+              class="has-focus custom"
+              label="Email"
+              error-text="Please enter a valid email"
+              helper-text="Enter an email"
+              counter="true"
+              maxlength="20"
+            ></ion-textarea>
+          </div>
+
+          <div class="grid-item">
+            <h2>Valid, Focus</h2>
+            <ion-textarea
+              label-placement="start"
+              class="ion-valid has-focus custom"
+              label="Email"
+              error-text="Please enter a valid email"
+              helper-text="Enter an email"
+              counter="true"
+              maxlength="20"
+            ></ion-textarea>
+          </div>
+
+          <div class="grid-item">
+            <h2>Invalid</h2>
```

Comment:
For some reason, the height isn't being changed for the invalid textarea when running locally. The same is happening for the others. No issues in the screenshots though.

![Screenshot 2024-03-04 at 5 05 38 PM](https://github.com/ionic-team/ionic-framework/assets/13530427/259e6a20-0deb-4ecb-af52-2316f36b799f)

Reply:
See my comment here: https://github.com/ionic-team/ionic-framework/pull/29090/files#r1518139609